readme.md

```diff
@@ -298,6 +298,7 @@
 
 ### Web frameworks
 
+- [Node.js Framework Overview](http://nodeframework.com/) - Hand-picked registry of Node.js frameworks.
```

Comment:
I already passed on this: https://github.com/sindresorhus/awesome-nodejs/pull/369#issuecomment-146457416